interface-spec.md

```diff
@@ -66,15 +68,15 @@
 
 **Properties:**
 
-- `Node .graph` the named graph, which is an IRI, a BlankNode or NodeVariable.
+- `Node .why` the provenence, which may be a named node represenitng a document on the web which has this graph, or a BlankNode or NodeVariable. Also extednable for the results of infreence.
```

Comment:
`.graph` would be an IRI (named node). So given the following N-Quads:
```
<a> <b> <c> <d>.
```
the object would be
```
{ subject: a, predicate: b, object: c, graph: d }
```
where `a`, `b`, `c`, `d` are IRIs (named nodes). In particular, `d` is _not_ a graph object.